app/src/sharedTest/java/org/oppia/android/app/topic/revisioncard/RevisionCardFragmentTest.kt

```diff
@@ -119,7 +118,6 @@ class RevisionCardFragmentTest {
   }
 
   @Test
-  @DisableAccessibilityChecks // TODO(#3251): Enable AccessibilityChecks
```

Comment:
Fixed because we mentioned minHeight `48dp` in toolbar text.

Reply:
<img width="1425" alt="Screenshot 2021-06-24 at 2 25 44 AM" src="https://user-images.githubusercontent.com/9396084/123171167-0f7f0700-d499-11eb-9ec9-d12620e5d9ac.png">